src/java/org/apache/cassandra/db/rows/LazilyInitializedUnfilteredRowIterator.java

```diff
@@ -105,4 +105,4 @@ public void close()
         if (iterator != null)
             iterator.close();
     }
-}
+}
```

Comment:
nit: necessary to remove? I have no idea if we have a style-guide thing for EoF newlines :D